docs/displaying-ads.mdx

```diff
@@ -2,6 +2,65 @@
 
 The AdMob package allows you to display three types of adverts; Interstitial, Rewarded & Banner.
 
+## App Open Ads
+
+App open ads are a special ad format intended for publishers wishing to monetize their app load screens.
+App open ads can be closed by your users at any time. App open ads can be shown when users bring your app to the foreground.
+
+App open ads are shown when your application launches or when users bring your application to the foreground.
+To make sure you have an ad ready to display when a user opens your app, you'll want to have a reference to an ad ready to use.
+
+That means you must preload a app open ad before you need to show the ad. That way, your app open ad is ready to show the next time the app is opened.
+
+To create a new app open ad, call the `createForAdRequest` method from the `AppOpenAd` class. The first argument
+of the method is the "Ad Unit ID". For testing, we can use a Test ID, however for production the ID from the
+Google AdMob dashboard under "Ad units" should be used:
+
+```js
+import { AppOpenAd, TestIds, AdEventType } from '@invertase/react-native-google-ads';
```

Comment:
```suggestion
import { AppOpenAd, TestIds, AdEventType } from 'react-native-google-mobile-ads';
```

Reply:
You could also clone the `Test` implementations in `example/App.tsx` to make a new one here for app open perhaps? Although...no idea how that will look (can you show them on a non-app open event?), would be good to add a test if possible